package.json

```diff
@@ -64,14 +61,15 @@
     },
     "browserslist": {
         "production": [
-            ">0.2%",
+            ">0.5%",
             "not dead",
-            "not op_mini all"
+            "not op_mini all",
+            "not IE 11"
         ],
         "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
+            "last 2 chrome version",
```

Comment:
Why are `development` and `production` different here and why are we changing it from last version to last 2 versions?

Reply:
Development builds are run on the latest browser for quicker testing.